### ISSUE TITLE
how to install on shared hosting

### ISSUE THREAD

**Issue by Olastech** [cmd3BOT/Luminosity#1]
how to i install it on shared hosting

**Comment by taras-budzyn** [comment]
> how to i install it on shared hosting

You can use the Setup locally section with some modification

[Setup locally](https://github.com/cmd3BOT/Luminosity#setup-locally)

Like, you should download code as zip and upload via FTP to shared hosting, also you should use shared hosting with apache web server, as this project use .htaccess files. All else should be similar to instructions.

**Comment by Olastech** [comment]
yes i have done that
![erro](https://user-images.githubusercontent.com/62516325/116684019-4c061900-a9a8-11eb-8358-877d41ece7cf.jpg)

**Comment by cmd05** [comment]
> yes i have done that
> ![erro](https://user-images.githubusercontent.com/62516325/116684019-4c061900-a9a8-11eb-8358-877d41ece7cf.jpg)

Are you using windows?

If not:
Yes that was an issue my bad. Git doesnt recognize change in case when renaming folders/files. So the folders and all files in ``app/Views/*`` should be lowercase. I renamed them but git wasnt recognizing. So right now it is loading ``info/index`` [Line 14](https://github.com/cmd3BOT/Luminosity/blob/master/app/Controllers/ViewControllers/Info.php#L14) but the actual folder in Views is named ``Info`` (with a capital ``I``). Do rename all files and folders in Views to lowercase, but not the Views folder itself.

I'll be fixing this soon.

**Edit: Fixed**
